src/metaswitch/clearwater/config_manager/etcd_synchronizer.py

```diff
@@ -62,8 +65,13 @@ def main(self):
                 break
 
             if value:
+                _log.info("Got new config value from etcd - filename {}, file size {}, MD5 hash {}".format(
+                    self._plugin.file(),
+                    len(value),
+                    md5(value).hexdigest()))
                 _log.debug("Got new config value from etcd:\n{}".format(value))
```

Comment:
We don't need the second debug log here, and I think the new log should be at debug level